.github/workflows/sync-labels.yml

```
@@ -0,0 +1,14 @@
+name: Sync Labels
+
+on:
+  push:
+    branches: [main]
+
```

Should it not have something like this, so that it only runs when a new port is added?
```suggestion
     paths:
      - .github/labels.yml
```